converter/biostudies.py

```diff
@@ -0,0 +1,176 @@
+from json_converter.json_mapper import JsonMapper
+from json_converter.post_process import default_to
+
+PROJECT_SPEC_BASE = [
+    {
+        'name': ['', default_to, 'Project Core - Project Short Name'],
+        'value': ['content.project_core.project_short_name']
+    },
+    {
+        'name': ['', default_to, 'HCA Project UUID'],
+        'value': ['uuid.uuid']
+    },
+    {
+        'name': ['', default_to, 'ReleaseDate'],
+        'value': ['releaseDate']
+    }
+]
+
+PROJECT_SPEC_SECTION = {
+    "accno": ['', default_to, "PROJECT"],
+    "type": ['', default_to, "Study"],
+    "attributes": ['$array', [
+            {
+                "name": ['', default_to, "Title"],
+                "value": ['content.project_core.project_title']
+            },
+            {
+                "name": ['', default_to, "Description"],
+                "value": ['content.project_core.project_description']
+            }
+        ],
+        True
+    ]
+}
+
+
+def array_to_string(*args):
+    value = ", ".join(args[0])
+    return value
+
+
+PROJECT_SPEC_PUBLICATIONS = {
+    "type": ['', default_to, "Publication"],
+    "$on": 'publications',
+    "attributes": ['$array', [
+            {
+                "name": ['', default_to, "Authors"],
+                "value": ['authors', array_to_string]
+            },
+            {
+                "name": ['', default_to, "Title"],
+                "value": ['title']
+            },
+            {
+                "name": ['', default_to, "doi"],
+                "value": ['doi']
+            },
+            {
+                "name": ['', default_to, "URL"],
+                "value": ['url']
+            }
+        ],
+        True
+    ]
+}
+
+
+def _parse_name(*args):
+    full_name = args[0]
+    position = args[1]
+
+    if not full_name:
+        return None
+
+    name_element = full_name.split(',', 2)[position]
+    return name_element[0] if name_element and position == 1 else name_element
+
+
+PROJECT_SPEC_AUTHORS = {
+    "type": ['', default_to, "Author"],
+    "$on": 'contributors',
+    # '$filter': ['project_role', _is_not_wrangler],
```

Comment:
Is there something missing here or should this be removed?

Reply:
I am going to remove it for now and add it if we need it. Tx for the notification.

Reply:
Done